src/licenses.rs

```diff
@@ -366,47 +366,127 @@ pub fn is_license_compatible(
     log(
         LogLevel::Info,
         &format!(
-            "Checking if license {dependency_license} is compatible with project license {project_license}"
+            "Checking if dependency license {dependency_license} is compatible with project license {project_license}"
         ),
     );
 
-    // Define a compatibility matrix using a HashMap
+    // Define what dependency licenses can be included in each project license
     let compatibility_matrix: HashMap<&str, Vec<&str>> = [
```

Comment:
Although I'm not familiar with the code base, shouldn't this `HashMap` contain an entry for `AGPL-3.0` as well, listing the licenses that can be used in dependencies?

Reply:
Yes, you're right. Was gonna move to a dynamic function. This is supposed to be intermediate fixed list to go through. So, added most commons ones for now in a hashmap.

Will add `AGPL-3.0` as well.